content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and software maintenance to paid customers. Supported software products include:
+
+* [UXP](https://www.upbound.io/product/universal-crossplane)
+* [Official Providers](https://marketplace.upbound.io/providers?tier=official)
+
+## Maintenance and Updates
+
+Upbound supports Generally Available releases for up to 12 months. Eligible code-fixes are provided via a new minor release on top of the latest major release branch for up to 3 releases from the most current major release. All software follows semantic versioning of `X.Y.Z`. A major release is a change in the first or second digit. A minor release is a change in the third digit.
+
+### UXP Schedule
+
+UXP follows the [Crossplane Release Cycle](https://docs.crossplane.io/knowledge-base/guides/release-cycle/). UXP has a major release every 3 months. Minor patch release are available as needed. Upbound provides support for the current + 3 previous major releases, for a minimum of 12 calendar months of coverage for a given release.
```

Comment:
If we commit to supporting current + 3 previous releases, then we are supporting more than what the XP project itself supports, which is only 3 releases. 

From the release-cycle page:
>This results in four releases per year, with the most recent three releases being maintained at any given time. When a new release is cut, the fourth most recent release reaches the end of life (EOL). Users can expect any given release to be maintained for nine months.

What this means is we would need to support a XP release that XP itself doesn't support. It's not impossible to do, but it could bring in all sorts of challenges. If this is what we want as a business we can make this work by shipping patches for unsupported XP release via UXP only rather than upstream as we do today.